Source/Core/DolphinWX/ISOFile.cpp

```diff
@@ -266,6 +208,37 @@ bool GameListItem::ReadPNGBanner(const std::string& path)
   return true;
 }
 
+void GameListItem::SetWxBannerFromRaw()
+{
+  // Need to make explicit copy as wxImage uses reference counting for copies combined with only
+  // taking a pointer, not the content, when given a buffer to its constructor.
+  if (!m_pImage.empty())
+  {
+    m_image.Create(m_ImageWidth, m_ImageHeight, false);
+    std::memcpy(m_image.GetData(), m_pImage.data(), m_pImage.size());
+  }
+}
+
+bool GameListItem::ReloadBannerIfNeeded()
+{
+  // Wii banners can only be read if there is a savefile,
+  // so sometimes caches don't contain banners. Let's check
+  // if a banner has become available after the cache was made.
+  if ((m_Platform == DiscIO::Platform::WII_DISC || m_Platform == DiscIO::Platform::WII_WAD) &&
```

Comment:
Sure, if that PR gets merged before this one. (If that happens, there's going to be a merge conflict due to that PR adding a call to `DiscIO::IsWii` in the old copy of this code, so it's going to be noticeable.)